.spelling

```diff
@@ -3,690 +3,387 @@
 # global dictionary is at the start, file overrides afterwards
 # one word per line, to define a file override use ' - filename'
 # where filename is relative to this configuration file
+# File Overrides:
```

Comment:
You sorted it too broadly.  There is a general section then file specific sections.  I suspect the PR will fail because now all the words say they are for `CODE_OF_CONDUCT.md`

Reply:
Ideally, words used in multiple files would be at the top in the `global` section and each file override has it's own specific section which is sorted.

Reply:
ah.. I misunderstood what was going on there. I will fix it.